GAME/CONTROL.C

```diff
@@ -1045,9 +1046,35 @@ void RemoveRoomFlipItems(struct room_info* r)//1F938(<), 1FB4C(<) (F)
 	}
 }
 
-void FlipMap(int FlipNumber)
+void FlipMap(int FlipNumber) // (F)
 {
-	S_Warn("[FlipMap] - Unimplemented!\n");
+	struct room_info* r = room;
+	for(int i = 0; i < number_rooms; i++, r++)
+	{
+		if (r->flipped_room >= 0 && r->FlipNumber == FlipNumber)
+		{
+			struct room_info temp;
```

Comment:
Must be declared at the very beginning of method.

Reply:
c89 only wants at the beginning of scope which is the case here
\+ tested with gcc -pedantic